ocaml/runtime/backtrace_nat.c

```diff
@@ -164,17 +164,19 @@ static debuginfo debuginfo_extract(frame_descr* d, int alloc_idx)
 {
   unsigned char* infoptr;
   uint32_t debuginfo_offset;
+  uint32_t frame_size;
 
   /* The special frames marking the top of an ML stack chunk are never
      returned by caml_next_frame_descriptor, so should never reach here. */
-  CAMLassert(d->frame_size != 0xffff);
+  CAMLassert(d->frame_size != 0xFFFF);
```

Comment:
I am surprised this does not need
to be updated (beside the case).

Reply:
This is a different value, bigger than the maximal length of the short frame. 
See `Emitaux.frame_size_long` and  `LONG_FRAME = 0x7FFF` in `roots_nat.c`.